src/components/organisms/AdvancedLendingOrderCard/AdvancedLendingOrderCard.tsx

```
@@ -100,13 +99,28 @@ export function AdvancedLendingOrderCard({
+
     const { address, isConnected } = useAccount();
 
     const [sliderValue, setSliderValue] = useState(0.0);
 
     const balanceRecord = useBalances();
     const isTablet = useBreakpoint('laptop');
 
+    const midPrice = useMemo(() => {
+        if (
+            orderBook.data &&
+            orderBook.data.borrowOrderbook &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)